aries-site/src/examples/components/layouts/LayerExample.js

```diff
@@ -0,0 +1,55 @@
+import React, { useContext, useState } from 'react';
+import { Button, Box, Heading, Layer, ResponsiveContext, Text } from 'grommet';
+import { Close } from 'grommet-icons';
+
+export const LayerExample = () => {
+  const [open, setOpen] = useState(false);
+  const size = useContext(ResponsiveContext);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+  return (
+    <>
+      <Box align="start">
+        <Button
+          color="blue"
+          label="Show me the Layer"
+          onClick={onOpen}
+          primary
+        />
+      </Box>
+      {open && (
+        <Layer
+          position="right"
+          full={size !== 'small' ? 'vertical' : true}
+          modal
+          onClickOutside={onClose}
+          onEsc={onClose}
+        >
+          <Box
```

Comment:
Why is the layer here presenting with a white background? I would think it would be "background-front".

Reply:
Background front would work.

Reply:
From Slack: Running into a situation where the Layer was coming through as white because it thinks it's on a light background -- trying to figure out how to get it to know that the whole site is in dark mode